pyqtgraph/opengl/items/GLTextItem.py

```diff
@@ -0,0 +1,90 @@
+
+        ====================  ==================================================
+        **Arguments:**
+        ------------------------------------------------------------------------
+        pos                   (3,) array of floats specifying text location.
+        color                 (4,) array of floats (0.0-1.0) or
+                              tuple of floats specifying
+                              a single color for the entire item.
+        text                  String to display.
+        font                  GLUT font. (Default: GLUT_BITMAP_HELVETICA_18)
+        ====================  ==================================================
+        """
+        args = ['pos', 'color', 'text', 'font']
+        for k in kwds.keys():
+            if k not in args:
+                raise Exception('Invalid keyword argument: %s (allowed arguments are %s)' % (k, str(args)))
+        for arg in args:
+            if arg in kwds:
+                value = kwds[arg]
+                if arg == 'pos':
+                    if isinstance(value, np.ndarray):
+                        if value.shape != (3,):
+                            raise Exception('"pos.shape" must be (3,).')
+                    elif isinstance(value, (tuple, list)):
+                        if len(value) != 3:
+                            raise Exception('"len(pos)" must be 3.')
+                elif arg == 'color':
+                    if isinstance(value, np.ndarray):
+                        if value.shape != (4,):
+                            raise Exception('"color.shape" must be (4,).')
+                    elif isinstance(value, (tuple, list)):
+                        if len(value) != 4:
+                            raise Exception('"len(color)" must be 4.')
+                elif arg == 'font':
+                    if value not in [
+                        GLUT_BITMAP_8_BY_13,
+                        GLUT_BITMAP_9_BY_15,
+                        GLUT_BITMAP_TIMES_ROMAN_10,
+                        GLUT_BITMAP_TIMES_ROMAN_24,
+                        GLUT_BITMAP_HELVETICA_10,
+                        GLUT_BITMAP_HELVETICA_12,
+                        GLUT_BITMAP_HELVETICA_18
+                    ]:
+                        raise Exception('"font" must be ' \
+                            '"GLUT_BITMAP_8_BY_13", ' \
+                            '"GLUT_BITMAP_9_BY_15", ' \
+                            '"GLUT_BITMAP_TIMES_ROMAN_10", ' \
+                            '"GLUT_BITMAP_TIMES_ROMAN_24", ' \
+                            '"GLUT_BITMAP_HELVETICA_10", ' \
+                            '"GLUT_BITMAP_HELVETICA_12", or ' \
+                            '"GLUT_BITMAP_HELVETICA_18".')
+                setattr(self, arg, value)
+        self.update()
+    
+    def paint(self):
+        if len(self.text) < 1:
+            return
+        self.setupGLState()
+
+        glColor4d(float(self.color[0]), float(self.color[1]), float(self.color[2]), float(self.color[3]))
```

Comment:
with the above change, we should be able to replace this with

```python
glColor4d(*self.color)
```

Reply:
I fixed in [97e39cd](https://github.com/pyqtgraph/pyqtgraph/pull/1776/commits/97e39cde831c710998d9cf8b02064144a1986cf9).